examples/ogl_tutorials/tutorial03_matrices/src/maths.ads

```diff
@@ -0,0 +1,34 @@
+
+with Ada.Numerics;
+with Ada.Numerics.Generic_Elementary_Functions;
+
+with GL.Types;
+
+package Maths is
```

Comment:
OK. Understood.
I would only have declared these functions because of some compilation complaint and certainly would have preferred not to declare them if I had realized an alternative solution at the time.
I suspect I initially declared the subtypes as new types but subsequently changed them to subtypes to solve some other problem.
Based on your advice, I'll remove the mathematical functions and solve any problems that arise without recourse to them.
The subtypes are provided for consistency with types in a set of tutorials.
If you prefer I can remove them.
I need to retain the package to declare the two procedures. (Unless there are equivalent OpenGLAda procedures that I haven't discovered.)
I'll also remove the unnecessay "private" which previously escaped my attention.